src/chocolatey.resources/helpers/functions/Get-WmiObject.ps1

```diff
@@ -0,0 +1,15 @@
+﻿## Get-WmiObject
+##############################################################################################################
+## Wrapper function. Replaces "Win32_Product" class with Get-Installer helper.
+##############################################################################################################
+
+function Get-WmiObject {
```

Comment:
That's awesome. Wish I would have thought about that with Write-Host, Write-Error and the like. :)